### ISSUE TITLE
Refine the concourse ci pipeline and setup testing ops man environments to cover different scenarios

### ISSUE THREAD

**Issue by steven-zou** [vmware/harbor#3772]
config with local db
config with LDAP
config with self-sign certs

**Comment by reasonerjt** [comment]
@steven-zou Can this be done in 0.8 timeframe?  we are very close now.

**Comment by steven-zou** [comment]
Ongoing, 1 day effort left. Maybe close to end today. 

config with local db DONE!
config with LDAP  DONE!
config with self-sign certs 80% DONE!

**Comment by steven-zou** [comment]
self-signed cert blocked by issue #3823 , postponed to next SP (24)

**Comment by steven-zou** [comment]
All scenarios(localdb, ldap and self-signed cert) are done! See pipeline https://10.40.53.21/teams/main/pipelines/habo-ci?groups=vmware-harbor-registry-ci
